src/locale/locales/de/messages.po

```diff
@@ -2086,71 +2015,67 @@ msgstr "Text-Eingabefeld"
 
 #: src/view/com/profile/ProfileHeader.tsx:306
 msgid "The account will be able to interact with you after unblocking."
-msgstr "Das Konto kann nach der Entsperrung mit Ihnen interagieren."
+msgstr ""
 
 #: src/view/screens/CommunityGuidelines.tsx:36
 msgid "The Community Guidelines have been moved to <0/>"
-msgstr "Die Gemeinschaftsrichtlinien wurden nach <0/> verschoben."
+msgstr "Die Community-Richtlinien wurden nach <0/> verschoben"
 
 #: src/view/screens/CopyrightPolicy.tsx:33
 msgid "The Copyright Policy has been moved to <0/>"
-msgstr "Die Urheberrechtspolitik wurde nach <0/> verschoben."
+msgstr ""
 
 #: src/view/com/post-thread/PostThread.tsx:433
 msgid "The post may have been deleted."
-msgstr "Der Beitrag kann gelöscht worden sein."
+msgstr "Möglicherweise wurde der Post gelöscht."
 
 #: src/view/screens/PrivacyPolicy.tsx:33
 msgid "The Privacy Policy has been moved to <0/>"
-msgstr "Die Datenschutzrichtlinie wurde nach <0/> verschoben."
+msgstr "Die Datenschutzerklärung wurde nach <0/> verschoben"
 
 #: src/view/screens/Support.tsx:36
 msgid "The support form has been moved. If you need help, please<0/> or visit {HELP_DESK_URL} to get in touch with us."
-msgstr "Das Support-Formular wurde verschoben. Wenn Sie Hilfe benötigen, wenden Sie sich bitte an<0/> oder besuchen Sie {HELP_DESK_URL}, um mit uns in Kontakt zu treten."
+msgstr "Das Support-Formular wurde verschoben. Wenn du Hilfe benötigst, wende dich bitte an<0/> oder besuche {HELP_DESK_URL}, um mit uns Kontakt aufzunehmen."
 
 #: src/view/screens/TermsOfService.tsx:33
 msgid "The Terms of Service have been moved to"
-msgstr "Die Allgemeinen Geschäftsbedingungen wurden verschoben nach"
+msgstr ""
 
 #: src/view/com/util/ErrorBoundary.tsx:35
 msgid "There was an unexpected issue in the application. Please let us know if this happened to you!"
-msgstr "Es gab ein unerwartetes Problem in der Anwendung. Bitte teilen Sie uns mit, wenn dies bei Ihnen der Fall ist!"
```

Comment:
Since we use informal "du" everywhere else, rather make it `"Es gab ein unerwartetes Problem mit der Anwendung. Bitte teile uns mit, wenn das bei dir der Fall ist!"`